Automattic/jetpack
# Search: Fix notice issue with new widget editor

#### Changes proposed in this Pull Request:

Fixes a notice issue when viewing a Jetpack Search widget in the new Widgets editor in WordPress 5.8 with `WP_DEBUG` enabled.


Before|After
------|-----
<img width="1638" alt="Screen Shot 2021-07-23 at 4 24 11 PM" src="https://user-images.githubusercontent.com/4044428/126847009-b5b94d4e-4fb2-4815-aab9-04a4b9bd7133.png">|<img width="1638" alt="Screen Shot 2021-07-23 at 4 25 53 PM" src="https://user-images.githubusercontent.com/4044428/126847125-c8da2b9d-d095-4b56-ab17-a8bab3c80fde.png">

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Spin up a Jurassic Ninja instance with `WP_DEBUG`, Jetpack Search (e.g. via Jetpack Complete), and Jetpack Beta (to switch to this branch).
* Navigate to `/wp-admin/widgets.php`. Ensure that there's a Jetpack Search widget and that no notice is shown within the widget block.
